Cargo.toml

```diff
@@ -45,7 +45,7 @@ once_cell = "1.16.0"
 pin-project-lite = "0.2.9"
 rand = "0.8.5"
 reqwest = { version = "0.12.4", default-features = false }
-ruma = { version = "0.11.1", features = [
+ruma = { git = "https://github.com/ruma/ruma.git", rev = "35fda7f2f7811156df2e60b223dbf136fc143bc8", features = [
```

Comment:
Potentially stating the obvious here, but if you wanna do a crates.io release soon (I think this was planned), make sure to only merge this PR afterwards 😉

Reply:
Oh. Well I don't think we'll have the Ruma change blocking this PR out on crates.io anytime soon.

Reply:
No rush in my side, it can wait.

Feel free to review in the meantime, so we can merge when we got a ruma release out.

Reply:
Alright, the policy changed back, could you please rebase to pull in #4371, this will get rid of the CI error.